src/FixedDataTable.js

```diff
@@ -847,17 +864,61 @@ var FixedDataTable = React.createClass({
       bodyHeight = totalHeightNeeded - totalHeightReserved;
     }
 
+    const fixedColumnGroups = [];
```

Comment:
Ugly transforms to extract data into a row consumable format.  I need to figure out if this can efficiently be merged with the result of convertColumnElementsToData, but I think if I did that, I'd have to pass the elementTemplates into the reducer